video2dataset/data_reader.py

```diff
@@ -57,7 +57,15 @@ def handle_url(url):
     
     return file.name, file, name
 
+class VideoDataReader:
+    def __init__(self) -> None:
+        pass
 
-class Downloader:
-  def __init__(self):
-    pass
+    def __call__(self, row, timeout, retries):
+        key, url = row
+        file_name, file, _ = handle_url(url)
```

Comment:
the last return value for handle_url is supposed to be the name you wish to save the video to. So for example sometimes I used this to save YouTube videos as {youtube_id}.ext but I guess maybe we should get rid of it altogether and just commit to key being the save name.

Reply:
yeah key should def not be that

but we can totally store the youtube_id and other things in metadata

Reply:
it's one of those things that was helpful for debugging but here it's definitely unnecessary, let's remove it